sentry.gradle

```
@@ -165,7 +165,8 @@ def extractReleasesInfo() {
 
         variant.outputs.each { output ->
             def versionCode = output.getVersionCode()
-            def releaseName = "${variant.getApplicationId()}@${variant.getVersionName()}+${versionCode}"
+            def defaultReleaseName = "${variant.getApplicationId()}@${variant.getVersionName()}+${versionCode}"
+            def releaseName = System.getenv("SENTRY_RELEASE_NAME") ?: defaultReleaseName
```

```suggestion
            def releaseName = System.getenv("SENTRY_RELEASE") ?: defaultReleaseName
```
Let's call it `SENTRY_RELEASE`.

---

done